<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>588</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>588</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>picCoder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/about.png</normaloff>resources/about.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="picImageFrame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>571</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>551</width>
       <height>521</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="picLayout">
      <item>
       <widget class="QLabel" name="picImageLbl">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="picDetailsLbl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="controlLayout">
        <item>
         <widget class="QPushButton" name="getEmbeddedDataBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveCodedImage"/>
    <addaction name="actionExportConversation"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuEmbed">
     <property name="title">
      <string>Embed...</string>
     </property>
     <addaction name="actionEmbedFile"/>
     <addaction name="actionEmbedConversation"/>
     <addaction name="separator"/>
     <addaction name="actionIncludePassword"/>
    </widget>
    <addaction name="actionStartConversation"/>
    <addaction name="menuEmbed"/>
    <addaction name="actionPreviewImage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUserGuide"/>
    <addaction name="actionChangeLog"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionChangeLog">
   <property name="text">
    <string>Change Log</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionEmbedFileg">
   <property name="text">
    <string>Embed File</string>
   </property>
  </action>
  <action name="actionSaveCodedImage">
   <property name="text">
    <string>Save picCoded Image</string>
   </property>
  </action>
  <action name="actionStartConversation">
   <property name="text">
    <string>Start Conversation</string>
   </property>
  </action>
  <action name="actionEmbedConversationg">
   <property name="text">
    <string>Embed Conversation</string>
   </property>
  </action>
  <action name="actionEmbedConversation_2">
   <property name="text">
    <string>Embed Conversation</string>
   </property>
  </action>
  <action name="actionPreviewImage">
   <property name="text">
    <string>Preview Image</string>
   </property>
  </action>
  <action name="actionEmbedFile">
   <property name="text">
    <string>Embed File</string>
   </property>
  </action>
  <action name="actionEmbedConversation">
   <property name="text">
    <string>Embed Conversation</string>
   </property>
  </action>
  <action name="actionIncludePassword">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Embed with Password</string>
   </property>
  </action>
  <action name="actionZipEmbedding">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zip before Embedding</string>
   </property>
  </action>
  <action name="actionExportConversation">
   <property name="text">
    <string>Export Conversation</string>
   </property>
  </action>
  <action name="actionUserGuide">
   <property name="text">
    <string>User Guide</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
